## `nixpkgs-review` result                                                                                                                                       
                                                                                                                                                                 
Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).                                                                                     
                                                                                                                                                                 
Command: `nixpkgs-review pr 355989`                                                                                                                              
                                                                                                                                                                 
---                                                                                                                                                              
### `x86_64-linux`                                                                                                                                               
<details>                                                                                                                                                        
  <summary>:x: 2 packages failed to build:</summary>                                                                                                             
  <ul>                                                                                                                                                           
    <li>openresty</li>                                                                                                                                           
    <li>openresty.doc</li>                                                                                                                                       
  </ul>                                                                                                                                                          
</details>                                                                                                                                                       
<details>                                                                                                                                                        
  <summary>:white_check_mark: 17 packages built:</summary>                                                                                                       
  <ul>                                                                                                                                                           
    <li>angie</li>                                                                                                                                               
    <li>angie.doc</li>                                                                                                                                           
    <li>angieQuic</li>                                                                                                                                           
    <li>angieQuic.doc</li>                                                                                                                                       
    <li>devpi-client</li>                                                                                                                                        
    <li>devpi-client.dist</li>                                                                                                                                   
    <li>devpi-server</li>                                                                                                                                        
    <li>devpi-server.dist</li>                                                                                                                                   
    <li>nginx (nginxStable)</li>                                                                                                                                 
    <li>nginx.doc (nginxStable.doc)</li>                                                                                                                         
    <li>nginxMainline</li>                                                                                                                                       
    <li>nginxMainline.doc</li>                                                                                                                                   
    <li>nginxQuic</li>                                                                                                                                           
    <li>nginxQuic.doc</li>                                                                                                                                       
    <li>nginxShibboleth</li>                                                                                                                                     
    <li>nginxShibboleth.doc</li>                                                                                                                                 
    <li>tests.testers.lycheeLinkCheck.network</li>                                                                                                               
  </ul>                                                                                                                                                          
</details>

Looks like this breaks OpenResty:

```
./configure: error: the HTTP rewrite module requires the PCRE library.
```

We can probably backport the upstream fix.

--- a/pkgs/servers/http/openresty/default.nix
+++ b/pkgs/servers/http/openresty/default.nix
@@ -10,12 +10,12 @@
 
 callPackage ../nginx/generic.nix args rec {
   pname = "openresty";
-  nginxVersion = "1.25.3";
-  version = "${nginxVersion}.2";
+  nginxVersion = "1.27.1";
+  version = "${nginxVersion}.1";
 
   src = fetchurl {
     url = "https://openresty.org/download/openresty-${version}.tar.gz";
-    sha256 = "sha256-LVZAIrBuM7Rfflz68eXcVx041hgDr5+idU3/81PCjZw=";
+    sha256 = "sha256-ebBx4nvcFD1fQB0Nv1BN5EIAcNhnU4xe3CVG0DUf1cA=";
   };
 
   # generic.nix applies fixPatch on top of every patch defined there.

